data/tribes/buildings/productionsites/atlanteans/well/init.lua

```diff
@@ -43,6 +43,9 @@ wl.Descriptions():new_productionsite_type {
          -- TRANSLATORS: Completed/Skipped/Did not start working because ...
          descname = _("working"),
          actions = {
+            -- time total: 20 + 20 + 3.6 = 43.6 sec
```

Comment:
Why no rounding here? :)

Reply:
If there are multiple output times (e.g. here 43.6 and 70.523 sec), I tend to not round it at all.
For wells, the second time is more interesting IMO, because wells will sooner or later degrade to slower speed.
And in the formula, number 10 is still questionable, so I would personally round it only after this is resolved.